File: examples/android/app/src/androidTest/java/com/mongodb/realm/examples/java/InitializeTest.java
package com.mongodb.realm.examples.java;

import com.mongodb.realm.examples.Expectation;
import com.mongodb.realm.examples.RealmTest;

import org.junit.Test;

import java.util.concurrent.TimeUnit;

import io.realm.mongodb.App;
import io.realm.mongodb.AppConfiguration;

import static com.mongodb.realm.examples.RealmTestKt.YOUR_APP_ID;

public class InitializeTest extends RealmTest {

    @Test
    public void testAccessAppClient() {
        // :code-block-start: access-app-client
        String appID = YOUR_APP_ID; // replace this with your App ID
        App app = new App(new AppConfiguration.Builder(appID).build());
        // :code-block-end:
    }

    @Test
    public void testAdvancedAccessAppClient() {
        // :code-block-start: advanced-access-app-client
        String appID = YOUR_APP_ID; // replace this with your App ID
        App app = new App(new AppConfiguration.Builder(appID)
                .appName("My App")
                .requestTimeout(30, TimeUnit.SECONDS)
                .build());
        // :code-block-end:
    }
}

File: examples/android/app/src/androidTest/java/com/mongodb/realm/examples/kotlin/InitializeTest.kt
package com.mongodb.realm.examples.kotlin

import com.mongodb.realm.examples.RealmTest
import com.mongodb.realm.examples.YOUR_APP_ID
import io.realm.mongodb.App
import io.realm.mongodb.AppConfiguration

import org.junit.Test
import java.util.concurrent.TimeUnit

class InitializeTest : RealmTest() {

    @Test
    fun testAccessAppClient() {
        // :code-block-start: access-app-client
        val appID : String = YOUR_APP_ID // replace this with your App ID
        val app: App = App(AppConfiguration.Builder(appID).build())
        // :code-block-end:
    }

    @Test
    fun testAdvancedAccessAppClient() {
        // :code-block-start: advanced-access-app-client
        val appID = YOUR_APP_ID // replace this with your App ID
        val app: App = App(AppConfiguration.Builder(appID)
            .appName("My App")
            .requestTimeout(30, TimeUnit.SECONDS)
            .build())
        // :code-block-end:
    }
}